src/ProgressOnderwijsUtils/Extensions/StringUtils.cs

```diff
@@ -107,9 +106,8 @@ public static string VervangRingelS([NotNull] string str, bool upper)
         public static string SepaTekenset([NotNull] string s)
             => SepaStripperRegexes.sepaStripper.Replace(s, "");
 
-        [CanBeNull]
         [Pure]
-        public static string SepaTekensetEnModificaties(string s)
+        public static string? SepaTekensetEnModificaties(string s)

```

Comment:
dit lijken me wel typisch methods om te gaan fixen. Deze is trouwens toch ook dan null if null(s)?